ProjectSolenodon/Assets/Scripts/GameManager.cs

```diff
@@ -0,0 +1,50 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using UnityEngine;
+
+/// <summary>
+/// The GameManager class controls the state of the game at any given moment.
+/// It makes use of the LevelManager class in order to build level scenes
+/// for the player.
+/// </summary>
+public class GameManager : MonoBehaviour

```

Comment:
Consider renaming this class.

"Manager" adds nothing to the name's descriptiveness. If it's to provide an interface for GameObjects to query state, why not call it GameState?